spin-wagi-http/Dockerfile

```diff
@@ -28,17 +28,19 @@ RUN set -xe; \
     tar xzvf ./wasi-sdk.tar.gz --strip-components 1 -C /opt/wasi-sdk/; \
     rm -rf ./wasi-sdk.tar.gz;
 
-FROM spin AS build
+FROM spin-devenv AS build
 
-COPY ./http-rust /app/http-rust
-WORKDIR /app/http-rust
-RUN cargo build --release
+COPY ./hello-cpp /app/hello-cpp
+COPY ./goodbye-rust /app/goodbye-rust
+COPY ./spin.toml /app/spin.toml
+
+WORKDIR /app
 
-COPY ./wagi-http-cpp /app/wagi-http-cpp
-WORKDIR /app/wagi-http-cpp
-RUN make build
+RUN spin build
 
 FROM scratch
 
-COPY --from=build /app /app
 COPY ./spin.toml /app/spin.toml
+COPY --from=build /app/hello-cpp/hello.wasm /app/hello-cpp/hello.wasm
+COPY --from=build /app/goodbye-rust/target/wasm32-wasi/release/goodbye.wasm /app/goodbye-rust/target/wasm32-wasi/release/goodbye.wasm
+
```

Comment:
Any reason for an extra blank line at the end?

Reply:
Fixed.